### PR TITLE
Started on binary install from cli

### PR DESCRIPTION
**Pull request by stamp**

### CODE REVIEW

File: stampzilla/clihandler.go [hunk 1]
```diff
@@ -11,11 +11,11 @@ import (
 	"text/tabwriter"
 
 	"github.com/codegangsta/cli"
+	"github.com/coreos/fleet/log"
```
Comment (jonaz) [inline stampzilla/clihandler.go]: Wrong log package?

File: stampzilla/installer/interface.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	"github.com/stampzilla/stampzilla-go/stampzilla/installer/source"
+)
+
+type Installer interface {
+	Prepare() error
+	Install(...string)
+	Update(...string)
+}
+
+func New(t string) (Installer, error) {
```
Comment (jonaz) [inline stampzilla/installer/interface.go]: t could be a new custom typ like 
```
type InstallSource uint8
const (
	InstallSourceBinaries = iota
	InstallSourceSourceCode
)
```

File: stampzilla/installer/interface.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	case "binaries":
+		return newFromBinaries(), nil
+	case "source":
+		return newFromSource(), nil
+	}
+
+	return nil, fmt.Errorf("No installer named \"%s\" is available", t)
+}
+
+func newFromBinaries() Installer {
```
Comment (jonaz) [inline stampzilla/installer/interface.go]: Those does not add much value :)

File: stampzilla/cli.go [hunk 4]
```diff
@@ -15,8 +15,17 @@ func main() {
 	app.Name = "stampzilla"
 	app.Version = VERSION + " (" + BUILD_DATE + ")"
 	app.Usage = "Manage stampzilla on the command line"
+	app.EnableBashCompletion = true
 
-	cliHandler := &cliHandler{installer.NewInstaller()}
+	logrus.SetLevel(logrus.InfoLevel)
+	for _, v := range os.Args {
+		if v == "-d" {
```
Comment (jonaz) [inline stampzilla/cli.go]: why both this and cli.BoolFlag u below?

File: stampzilla/installer/binary/releases.go [hunk 5]
```diff
@@ -0,0 +1,53 @@
+package binary
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"time"
+)
+
+type Release struct {
```
Comment (jonaz) [inline stampzilla/installer/binary/releases.go]: Can you put the releases stuff in own package? I think we might want to use that in multiple locations :)

File: stampzilla/installer/source/installer.go [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+	head, err = exec.LookPath(head)
+	if err != nil {
+		return "", err
+	}
+	cmd := exec.Command(head, parts...)
+	//cmd.Env = []string{"GOPATH=$HOME/go", "PATH=$PATH:$GOPATH/bin"}
+	cmd.Env = []string{
+		"GOPATH=/home/stampzilla/go",
+		"PATH=" + os.Getenv("PATH"),
+		"STAMPZILLA_WEBROOT=/home/stampzilla/go/src/github.com/stampzilla/stampzilla-go/nodes/stampzilla-server/public/dist",
```
Comment (jonaz) [inline stampzilla/installer/source/installer.go]: We need to think about this :)  or bake it into the binary...

**Comment by stamp** [comment]
Test failed because: Max difference between 2017-03-25 15:20:32.263310041 +0000 UTC and 2017-03-25 15:20:32.290362445 +0000 UTC allowed is 10ms, but difference was -27.052404ms

**Pull request merged**